AIDA 704:  Clusters must be able to contain Relations (#145)
NextCenturyCorporation/AIDA-Interchange-Format: src/main/resources/com/ncc/aif/ontologies/README.md

### InterchangeOntology

This ontology contains the Interchange vocabulary and specifies how the defined classes and properties may interact.

> It is defined in such a way that it should be able to be used with OWL reasoners.

Reviewer: ```suggestion
It is defined using strict OWL and should be able to be used with OWL reasoners.
```

Author: Changed.
